components/core/src/clp_s/search/sql/sql.cpp

```diff
@@ -0,0 +1,55 @@
+#include <any>
+#include <iostream>
+#include <memory>
+
+#include <antlr4-runtime.h>
+#include <spdlog/spdlog.h>
+
+#include "../antlr_common/ErrorListener.hpp"
+#include "../EmptyExpr.hpp"
+#include "../Expression.hpp"
+#include "SqlBaseVisitor.h"
+#include "SqlLexer.h"
+#include "SqlParser.h"
+
+using namespace antlr4;
+using namespace sql;
+using clp_s::search::antlr_common::ErrorListener;
```

Comment:
```suggestion
using antlr4::ANTLRInputStream;
using antlr4::CommonTokenStream;
using clp_s::search::antlr_common::ErrorListener;
using sql::SqlBaseVisitor;
using sql::SqlLexer;
using sql::SqlParser;
```
We shouldn't add using to the entire namespace; instead we should add using antlr4::xxx to refer to specific used symbols